app_radio_module/Kconfig

```diff
@@ -20,6 +20,12 @@ config LORA_TX_QUEUE_SIZE
 	help
 	  The number of LoRa packets that can be queued for transmission.
 
+config GNSS_DATA_SAMPLE_INTERVAL
+	int "GNSS data sample interval (seconds)"
+	default 1
+	help
+	  The interval at which GNSS data is sampled.
```

Comment:
Interval at which GNSS data is transmitted*
Its getting constantly sampled. We just choose when it gets put on a queue for transmission

Reply:
resolved https://github.com/RIT-Launch-Initiative/FSW/pull/85/commits/d4b8e90ed450f2c1fe9efe8b1d5f84ee75a0447b